### DISCUSSION TITLE
How to make Prettier add commas for objects

### DISCUSSION THREAD

**Discussion by erhanertem** [prettier/prettier#12673] [Q&A] [answered]
I can't get Prettier in JS to add commas in VScode automatically. Is it now and then a usual behaviour ? I searched stackoverflow and there are mentions of Prettier being able to set commas at the end of each object literal property dated back to year 2020. 

Note: Prettier puts comma only if one line of property is given and on-save the line receives comma. However, in the event of multiple properties, on-save none receives a comma.

const lufthansa = {
  airline: 'Lufthansa' , <-here
  iataCode: 'LH' , <-here
  bookings: [] ,<-here
};

**Comment by cerm88** [comment, marked as answer]
Commas must always be added by you, the last comma is optional and prettier adds it:

[Example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEAEnADYkQDqEATiQCYAUw+AOlPvgObVxy5Sd8AXlYgAFqXIsQAGlbsuPPnDrD8AcmlVadaerlsOAZxwkEMNagCGJI3AMLoAWQjo7AeQBucajLYBfAEp8YDZDfEgoIwsIACMAKzVmcI4oKwBbOCQNAGErWggjfRT8EitogH00zOz1ACUIOmocTnQ4AC9ihQ4rOia4IyNagDUbBDAcKzlhhA62ss0FfwBucPCcVAYAQm5efk5A4B4YdGp2KHQyFbDujgB6O-wAFXcAEXds1+h1Czc4fGoVigdAg6XwOHYPGBPnCZgsFzBIicVhgYgAdKhyDR8Axkai0YDgaCGMEAFT4ACMAFEANQAdkCaJgEAAyjBmgISQS4AAHMpgOAMO4sFhokV0Gl3FoGEDSQI3DjHU7sAA8dBwngiZUGADkMnAhOpiGRKDR6Op8LgYGYhMAAAYATVc+Hy-08OBMzOo+ARsR8+AAJMwEfh-Hb-PhnK4PN5qLao38vD5-AA+BUcfAq6LUaCcFPMXZ8CGcNFGEg4AUMfAABjkFPwjOZAFUeTyfHk7CT8DTFHti6Xy5WKY2IAAZCAAd3b5UFwX8Kru2dzaduwEL+zRCDoRgoOFRDGkfhAwQA-KJz9kszygfhogBPG3AYCQcjUWoi3Z39T+VOH0QLoxrygFNQxKFU4HSFcWBgAslHgVR-DCGAFwglcMxCHETDMWB5VuDgz2kMVZTw-BsmkLZpEQqAwLudVPBTVZqKgfxZBACAeVwaAjGQUB8hzCcAAVXSiZAQBsCcrDvbiZBAWJATAABrPgWX1UcITgZBrFsexZPkpSYBZa8JgEZB2TaGSIL9PoVFHIFWisTg4AAMRodIUX2USrHQZlWLEGB0hICgxD3AYjLgFkEE9DU9zvUSwEGViITsagYAEwFODczSbDsGT4iMAAPFlizMABFdAIHgLLtJk69qGS0TYisP0SFYnkORgXc6FRZAKWrWsQDawo4AoQEeVEtqBh8bxWIAR3K+A0vY7iUHKABaKBeDoFRWJ4OacB4NKHMypAtJykA7HSHBTOoczzuKuAyoqjSTuynSYCazruqQAAWGT2SsUxixyUFjpAAYAFZWL+J4muW06dM8NoAEloVgFkwGaDiAEFgTZB9nvhn8gA)
